src/site/content/en/metrics/inp/index.md

```diff
@@ -63,9 +63,7 @@ An interaction's latency consists of the single longest [duration](https://w3c.g
 For more details on how INP is measured, read the ["What's in an interaction?"](#whats-in-an-interaction) section.
 {% endAside %}
 
-## What's a "good" INP value?
```

Comment:
I have to reject this change, because it's important that people know that the reason why 200 milliseconds is the target is because of the varying capabilities of devices out there. Otherwise we will get questions like "why not 100 milliseconds?" while citing research that supports that conclusion.

Reply:
Isn't that the same [as for LCP](https://web.dev/defining-core-web-vitals-thresholds/#largest-contentful-paint) where the ideal time would be 1 second, but for achievability reasons we stretch this out to 2.5 seconds?

Reply:
Though maybe we should keep the caveat paragraph until such time that INP becomes a CWV and so can be added to that same note?

I still think we should remove the quotes around "good" for consistency.

Reply:
OK I added that paragraph back in for now.